components/faqs/nesting.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
+const sample = (`
+const ColorChanger = styled.section\`
+  background: papayawhip;
+
+  > h2 {
+    color: palevioletred;
+  }
+
+  @media(min-width: 768px) {
+    background: mediumseagreen;
+
+    > h2 {
+      color: papayawhip;
+    }
+  }
+\`;
+render(
+  <ColorChanger href="#">
+    <h2>Hello world!</h2>
+  </ColorChanger>
+)
+`).trim()
+
+const Nesting = () => (
+  <SectionLayout title="Nesting">
```

Comment:
> If you think about it, the following sections could also be regarded as "Tips & Tricks"

I did think about it, which is why I'm wondering whether these examples wouldn't be better with the same treatment. As much as anything, I'm not sure what the question here is! Is it "can I nest?" or is it broader: "What are the gaps with Sass"?